internal/client/discord/discord_parser.go

```diff
@@ -0,0 +1,16 @@
+package discord
+
+import (
+	"haracho-go/internal/client"
+	"strings"
+)
+
+type Parser struct {
+	rawMessage string
+	ctx        client.CommandContext
+}
+
+func (p Parser) Parse() (*client.Arg, client.CommandContext) {
+	split := strings.Split(p.rawMessage, " ")
+	return &client.Arg{CommandName: split[0], Args: split[1:]}, p.ctx
```

Comment:
```suggestion
	return client.NewArg(p.rawMessage), p.ctx
```
別記の通りに `Arg` を直すとこうなる